src/components/Home/View/index.js

```diff
@@ -54,15 +56,19 @@ const HomeView = (props) => {
         <FlatList
           keyExtractor={item => item.eventId.toString()}
           data={upcomingEvents}
-          renderItem={({ item }) => (
-            <ListItem
-              statusId={item.eventStatus.eventStatusId}
-              status={item.eventStatus.description}
-              startDate={item.start}
-              endDate={item.end}
-              duration={item.halfDay ? 0.5 : getDuration(item.start, item.end)}
-            />
-          )}
+          renderItem={({ item }) => {
+            const isWfh = (item.eventType.description === eventType.WFH);
+            return (
+              <ListItem
+                statusId={isWfh
```

Comment:
this is super petty, but just to keep the code pretty and readable can we go something like
```
statusId={isWfh ?
    holidayStatus.WFH :
    item.eventStatus.eventStatusId}
```

Reply:
ok, I'm surprised, but lets go with your esLint for each of the examples I've pointed out